source/fundamentals/crud/read-operations/specify-query.txt

```diff
@@ -0,0 +1,286 @@
+matching criteria in a **query filter**. Query filters contain one or more query
+operators to determine which documents to include in the resulting set.
+
+.. tip:: Find All Documents
+
+   Passing an empty query filter matches all documents in the collection.
+
+.. tip:: Using LINQ
+
+   You can also specify queries using LINQ. To learn more about using LINQ, see
+   :ref:`csharp-linq`.
+
+The examples in this guide use the following documents in a collection called
+``guitars``.
+
+.. code-block:: json
+ 
+   { "_id": 1, "make": "Fender", "models": ["Stratocaster", "Telecaster"], "establishedYear": 1946, "rating": 9 }
+   { "_id": 2, "make": "Gibson", "models": ["Les Paul", "SG", "Explorer"], "establishedYear": 1902, "rating": 8 }
+   { "_id": 3, "make": "PRS", "models": ["Silver Sky", "SE", "Custom"], "establishedYear": 1985, "rating": 9 }
+   { "_id": 4, "make": "Kiesel", "models": ["Ares", "Vader", "Solo"], "establishedYear": 2015 }
+   { "_id": 5, "make": "Ibanez", "models": ["RG", "AZ"], "establishedYear": 1957, "rating": 7 }
+   { "_id": 6, "make": "Strandberg", "models": ["Boden", "Salen"], "establishedYear": 1982 }
+
+The following ``Guitar`` class models the documents in this collection:
+
+.. literalinclude:: ../../../includes/fundamentals/code-examples/specify-query/Guitar.cs
+   :language: csharp
+   :copyable:
+   :dedent:
+
+Literal Values
+--------------
+
+Literal value queries return documents with an exact match to your search criteria.
+
+The following example specifies a query as a parameter to the ``Find()`` method,
+using the ``Guitar`` class. The query matches all documents where the ``make`` 
+field equals "Fender".
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: ../../../includes/fundamentals/code-examples/specify-query/FindEqPOCO.cs
+      :language:  csharp
+
+   .. output:: 
+      :language: none
+      :visible:
+
+      { "_id" : 1, "make" : "Fender", "models" : ["Stratocaster", "Telecaster"], "establishedYear" : 1946, "rating" : 9 }
+
+The following example creates a query filter using builders that matches all 
+documents where the ``make`` field equals "Fender".
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: ../../../includes/fundamentals/code-examples/specify-query/FindEqBuilder.cs
+      :language:  csharp
+
+   .. output:: 
+      :language: none
+      :visible:
+
+      { "_id" : 1, "make" : "Fender", "models" : ["Stratocaster", "Telecaster"], "establishedYear" : 1946, "rating" : 9 }
+
+
+To learn more about using builders, see :ref:`csharp-builders`.
+
+Comparison
+----------
+
+Comparison operators analyze the value in a document against the specified value
+in your match criteria. Common comparison operators include: ``>`` or ``Gt()`` 
+for "greater than", ``<=`` or ``Lte()`` for "less than or equal to", and ``!=``
+or ``Ne()`` for "not equal to".
+
+For a full list of comparison operators, see the :manual:`Comparison
+Query Operators </reference/operator/query-comparison/>` page.
```

Comment:
Is this a team convention, or a recent change? I know that we have been using `ref` in place of full links, but I didn't realize that we weren't using `manual` tags anymore? Just looking for clarification since I've been using `manual` tags in most of the pages I've been writing.

Reply:
This a convention used by most of the docs org now I think. We try to use `ref` whenever possible. The reason for this is that we as an org have tried to commit to never removing ref anchors, so they are generally fairly durable (where as we don't commit to changing URLs of pages).

Reply:
I can make a ticket to update these to add these refs since they don't currently exist, and then update them on this page.